recipes/mfast/all/test_package/message_printer.h

```diff
@@ -0,0 +1,139 @@
+#ifndef MESSAGE_PRINTER_H
```

Comment:
Please, try to use something simpler, small. The test package is used for checking the package, not to exercise the project. Code examples are goods most of time, but this one is too big, includes boost as external dependency ...

Reply:
Yeah, sorry for that. I forgot to trim this example down after I copied it from the examples. I'll be doing this tomorrow at lunch probably.

Reply:
I think it's passable now, please confirm...